spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature 'adding comments' do
+
+  let!(:sea) {Photo.create(title:'Awesome sea',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space missing after colon.